1.0/sdk/jessie/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # Install .NET Core SDK
-ENV DOTNET_SDK_VERSION 1.0.4
+ENV DOTNET_SDK_VERSION 1.1.0-preview1-005124
```

Comment:
The .NET Core docker versioning is in terms of the Runtime.  Think of it this way, if you want to build an app that targets 1.0, then this is the SDK image you should use to build your app.